thresher/admin.py

```diff
@@ -1,6 +1,21 @@
 from django.contrib import admin
 from thresher.models import *
+from django.contrib.auth.admin import UserAdmin
 
+class UserProfileInLine(admin.StackedInline):
```

Comment:
What is the meaning of InLine in this context?  Perhaps add a docstring to the class to explain.